docs/insert.rst

```diff
@@ -153,3 +154,241 @@ in as a ``json`` parameter, converted to a set of ``json`` objects with
       {title: 'Deadpool 3', release_year: null},
     ]
   });
+
+Here's a more complex example, demonstrating how to complete a nested insert
+with conflicts on the inner items. First, take a look at the schema for this
+database:
+
```

Comment:
Instead of duplicating this on both pages, maybe on this page it could just say that you can do bulk inserts with `e.for`, and link to the full example on the `for` page?